src/eigen.jl

```diff
@@ -6,12 +6,14 @@ end
 @inline function eigvals(a::StaticArray; permute::Bool=true, scale::Bool=true)
     if ishermitian(a)
         _eigvals(Size(a), Hermitian(a), permute, scale)
+    elseif Size(a) == Size{(1,1)}()
+        _eigvals(Size(a), a, permute, scale)
     else
         error("Only hermitian matrices are diagonalizable by *StaticArrays*. Non-Hermitian matrices should be converted to `Array` first.")
```

Comment:
Hmm, this error seems like an inconsistency with various other `StaticArrays` code (not your fault, obviously, nor something that needs to be fixed here unless you feel the urge.  But we should call `Base` and propagate the static size to the output rather than erroring out).